pkgs/applications/version-management/bit/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "bit";
+  version = "0.5.11";
+
+  src = fetchFromGitHub {
+    owner = "chriswalz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rdi6wsr5w64gvqkdysxrxfjxi3xbwigy3sk8wsx9zp4fql9xnw1";
+  };
+
+  subPackages = [ "." ];
+
+  vendorSha256 = "16c1mcvz3z17pixp904hazivx2n7y592nlx3ndnyajk7ljmnwfzy";
+
```

Comment:
Does it install completions? As is?

https://github.com/chriswalz/bit#using-go-harder-way-to-install

Could be nice if the derivations would install completion, but if no, no big deal.

Reply:
Not sure what the correct workflow for running this would be, bit has an interactive mode, so just made a self-contained nix.